src/dendropy/datamodel/basemodel.py

```diff
@@ -1204,6 +1204,11 @@ def add_citation(self,
         annotation : |Annotation|
             The new |Annotation| created.
         """
+
+        deprecate.dendropy_deprecation_warning(
+            message="Deprecated since DendroPy 5: BibTexEntry is broken. If this is important, open an issue on GitHub."
```

Comment:
```suggestion
            message=(
                "BibTexEntry is deprecated since DendroPy 5 and will be removed in a future release ."
                "It no longer maintaned, and known to be broken. "
                "If this functionality is needed, please open an issue on GitHub."
            ),
```